.gitignore

```diff
@@ -22,4 +22,5 @@ results
 npm-debug.log
 node_modules
 coverage
-/mocks/handlebars-component/_compile-static-package6/
\ No newline at end of file
+/mocks/handlebars-component/_compile-static-package6/
+package-lock.json
```

Comment:
I see that a `yarn.lock` made it through (packages/oc-s3-storage-adapter/yarn.lock), as we haven't yet started adding locks file in any of our repo, perhaps we can also remove the yarn lock file and ignore it as well here?